core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -157,8 +157,8 @@
   /**
    * Provides a custom {@link LoadBalancer.Factory} for the channel.
    *
-   * <p>If this method is not called, the builder will use {@link SimpleLoadBalancerFactory} for the
-   * channel.
+   * <p>If this method is not called, the builder will use {@link PickFirstLoadBalancerFactory}
```

Comment:
You missed this out :)
And sorry if I didn't make myself clear, but I suggested naming it to Dummy.